### PR TITLE
Enhance User Profile Display and Styling

### PR DESCRIPTION
**Pull request by kit0-0**
**Description:**
This pull request introduces several updates to the user profile display and styling. Key changes include the implementation of a custom design for user details, improved handling of user photos, and a refined presentation of the number of posts. Additionally, the styling enhancements have been applied consistently throughout the application for a more cohesive and visually appealing user interface. The goal is to provide a better user experience and maintain a modern and responsive design.

### CODE REVIEW

File: app/views/users/index.html.erb
Comment (fobadara) [inline app/views/users/index.html.erb:1]: [OPTIONAL] It is a good practice to use rails partials in scenarios like this. This helps to keep your code DRY.

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
+<% @recent_posts.each do |post| %>
+  <div class="custom-post custom-bio">
+    <h3>Post <%= raw "#" + post.id.to_s %></h3>
+    <p><%= post.text %></p>
+    <div class='custom-post-interactions'>
+      <span>Comments: <%= post.comments_counter %>,</span>
+      <span>Likes: <%= post.likes_counter %></span>
+    </div>
+  </div>
+<% end %>
```
Comment (fobadara) [inline app/views/users/show.html.erb]: [REQUIRED] You are expected to use the recent_post method created in the user model, not the variable created in your controller. Kindly fix this.

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @kit0-0, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

REDO

## Highlights
✅ Descriptive README
✅ Descriptive PR summary
✅ Clean code
✅ Good first attempt
✅ Good implementation of the principle of separating concerns.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @kit0-0, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Good job implementing the requested changes 💯 


## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**